tasks/main.yml

```diff
@@ -53,6 +53,11 @@
     is_prod: "{{ archivematica_src_environment_type == 'production' }}"
   tags: "always" # inocuous to use always here
 
+- name: "Include common-RHEL.yml for RHEL"
+  include: "common-RHEL.yml"
+  when: "ansible_distribution == 'RedHat'"
+  tags: "always" # inocuous to use always here
+
 - name: "Install necessary packages required by this role"
```

Comment:
I think it'd be better not to use tag "always" unless it's some task in the role that does not modify the target host (e.g., set facts, include_vars, etc), and instead using multiple tags related to the role (as the include common.yml task below). 
Tag "always" will execute even if running ansible with the -t config parameter, so for example if a playbook  includes other roles in addition to the archivematica role, and we would like to only execute the other roles and not the archivematica one, there is no way to prevent ansible executing these tasks.

Reply:
Is this role working ok with CentOS?  epel-release doesn't seem to be installed with CentOS (only with RedHat)

Reply:
Thanks Hector, I'll remove the always task.

About the CentOS question, it is working fine when running the task that uses the RedHat vars: https://github.com/artefactual-labs/ansible-archivematica-src/blob/stable/1.7.x/vars/RedHat.yml

But RHEL needs the rpm package from fedora.

Reply:
Great, thank you!